vitepress/.vitepress/theme/components/Duration.vue

```diff
@@ -0,0 +1,24 @@
+<script>
+const durationPropOption = {
+  validator: value => Number(value).toString() == value,
+  required: false,
+  default: 0,
+}
+</script>
+
+<script setup>
+import { duration } from 'datetime-attribute'
+
+const props = defineProps({
+  w: durationPropOption,
+  d: durationPropOption,
+  m: durationPropOption,
```

Comment:
Adding missing props (hours, seconds):

```suggestion
  h: durationPropOption,
  m: durationPropOption,
  s: durationPropOption,
```